bosh-release/setup.sh

```diff
@@ -1,314 +0,0 @@
-#!/bin/bash
```

Comment:
What would be the reason for removing this file?

Reply:
@imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

Reply:
The PR shows some files have been moved but some as deleted and added as new.